<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgDiscountCard</class>
 <widget class="QDialog" name="DlgDiscountCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="7" column="0" colspan="3">
      <widget class="FBtnOkCancel" name="widget" native="true"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Card code</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPLineEdit" name="leName">
       <property name="field" stdset="0">
        <string>NAME</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPLineEdit" name="leInfo">
       <property name="field" stdset="0">
        <string>INFO</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>State</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPLineEdit" name="leCode">
       <property name="field" stdset="0">
        <string>CODE</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPLineEdit" name="leId">
       <property name="field" stdset="0">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPComboBox" name="cbState">
       <property name="fieldId" stdset="0">
        <string>STATE_ID</string>
       </property>
       <property name="fieldName" stdset="0">
        <string>STATE_NAME</string>
       </property>
       <property name="table" stdset="0">
        <string>COSTUMERS_STATES</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Info</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Card value</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPLineEdit" name="leCardValue">
       <property name="field" stdset="0">
        <string>CARD_VALUE</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPComboBox</class>
   <extends>QComboBox</extends>
   <header>qpcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>FBtnOkCancel</class>
   <extends>QWidget</extends>
   <header location="global">fbtnokcancel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QPLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qplineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
